thoth/workflow_helpers/common.py

```diff
@@ -40,3 +53,32 @@ def store_messages(output_messages: list):
 
     if output_messages:
         _LOGGER.info(f"Successfully stored file with messages to be sent!: {output_messages}")
+
+
+def send_metrics(component_name: str):
+    """Send metrics to pushgateway."""
+    pushgateway_url = Configuration._THOTH_METRICS_PUSHGATEWAY_URL
+    deployment_name = Configuration._THOTH_DEPLOYMENT_NAME
+
+    prefix = "workflow-helpers"
```

Comment:
I think it's fine not to pass any component here, they are all using the same version of storages because they are all using the same image.  It should be enough to just alert when `workflow-helpers` alembic version is out of date.

Reply:
Thanks @KPostOffice I had doubt indeed on what was better from metrics point of view, but indeed we have one image of workflow-helpers usually.